build/cpython/Makefile

```diff
@@ -115,3 +115,4 @@ $(PYENV_DIR): /opt/redislabs $(CPYTHON_PREFIX)
 	export LANG=C.UTF-8; \
 	pipenv --site-packages install --python $(CPYTHON_PREFIX)/bin/python3
 	$(SHOW)cp $(CPYTHON_PREFIX)/Pipfile.lock .
+	cd $(PYENV_DIR);pipenv run pip install git+https://github.com/RedisGears/redisgears-py.git
```

Comment:
It's better to go:
cd $(PYENV_DIR); pipenv install git+https://github.com/RedisGears/redisgears-py.git#egg=redisgears-py
$(SHOW)cp $(CPYTHON_PREFIX)/Pipfile.lock .

Because otherwise installed package will not be included in Pipfile* files.

Reply:
Even better:
Add the following to the [packages] section in build/cpython/Pipfile:
redisgears-py = {git = "https://github.com/RedisGears/redisgears-py.git"}